packages/frontend/src/components/navbar/Navbar.js

```diff
@@ -50,45 +53,50 @@ function Navbar () {
 
   return (
     <Box px={3}>
-        <Flex
-          className={'Navbar'}
-          maxW='1980px'
-          ml='auto'
-          mr='auto'
-          h={16}
-          alignItems={'center'}
-          justifyContent={'space-between'}
-        >
-          <IconButton
-            size={'md'}
-            icon={isOpen ? <CloseIcon /> : <HamburgerIcon />}
-            aria-label={'Open Menu'}
-            display={{ lg: 'none' }}
-            onClick={isOpen ? onClose : onOpen}
-          />
+      <Flex
+        className={'Navbar'}
+        maxW='1980px'
+        ml='auto'
```

Comment:
Wrap the component parameters in {}, please.
`ml={'auto'}` etc.

Reply:
Is there a lint rule for that?